dashboard/app/views/layouts/_survey.html.haml

```diff
@@ -0,0 +1,67 @@
+-# If user is seeing English
+- if I18n.locale.to_s.downcase.split('-').first == "en"
+
+  -# If user hasn't already filled out this survey
+  - if !current_user[:properties]['survey2015_value']
```

Comment:
the serializedattributes stuff defines methods for you so you can use `current_user.survey2015_value`